src/lib/core/option/option.ts

```diff
@@ -220,6 +220,11 @@ export class MatOption implements AfterViewChecked, OnDestroy {
     }
   }
 
+  /** Returns the aria-selected value for the option. */
+  _getAriaSelected(): string|null {
+    return this.selected || this.multiple ? this.selected.toString() : null;
```

Comment:
Can you add a comment here that captures the context for this change? Something like
```txt
We explicitly omit the `aria-selected` attribute from single-selection, unselected options.
Including the `aria-selected="false"` attributes adds a significant amount of noise to
screen-reader users without providing useful information.
```

Reply:
You can skip the `toString` here. It'll be converted to a string when it eventually gets assigned to the attribute. Also you should be able to reduce it to this:

```ts
return this.selected || (this.multiple ? false : null);
```